templates/web/pages/product/product_page.tt.html

```diff
@@ -184,7 +184,7 @@ <h4>[% lang("nova_groups_s") %]
                 <a href="/nova" title="[% a_title %]">
                 [% display_icon('info') %]</a>
             </h4>
-            <a href="/nova" title="[% a_title %]"><img src="/images/misc/nova-group-[% group %].svg" alt="[% display %]" style="margin-bottom:1rem;max-width:100%"></a><br>
+            <a href="/nova" title="[% a_title %]"><img src="/images/attributes/nova-group-[% group %].svg" alt="[% display %]" style="margin-bottom:1rem;max-width:100%"></a><br>
```

Comment:
we are not very consistent… in donate_banner.tt we use static.openfoodfacts.org, there no. (not so important, but just to notice)

Reply:
Right, we should fix that in another PR.